tests/integration/challenges/utils.py

```diff
@@ -42,3 +43,15 @@ def generate_noise(noise_size) -> str:
             k=noise_size,
         )
     )
+
+
+def run_multiple_times(times):
```

Comment:
this challenge will have to be successful 3 times in a row, since it's underministic.

Reply:
Makes sense.